From e04f8a87714603c47d0b5ba58647475e3aa01581 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 3 May 2021 12:02:17 -0700
Subject: [PATCH] features.h: Bump __GLIBC_MINOR__ to 34

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 include/features.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/features.h b/include/features.h
index fcd1a9502f..8cbe2a10a7 100644
--- a/include/features.h
+++ b/include/features.h
@@ -475,7 +475,7 @@
 /* Major and minor version number of the GNU C library package.  Use
    these macros to test for features in specific releases.  */
 #define	__GLIBC__	2
-#define	__GLIBC_MINOR__	33
+#define	__GLIBC_MINOR__	34
 
 #define __GLIBC_PREREQ(maj, min) \
 	((__GLIBC__ << 16) + __GLIBC_MINOR__ >= ((maj) << 16) + (min))
